lib/lndclient/LndClient.ts

```diff
@@ -155,8 +146,8 @@ class LndClient extends SwapClient {
       this.credentials = grpc.credentials.createSsl(lndCert);
     } else {
       this.logger.error(`could not load tls cert from ${certpath}, is lnd installed?`);
-      await this.setStatus(ClientStatus.Misconfigured);
-      this.initRetryTimeout = global.setTimeout(this.init, 5000, awaitingCreate);
+      this.setStatus(ClientStatus.Misconfigured);
+      this.initRetryTimeout = setTimeout(this.init, 5000);
```

Comment:
Would it make sense to use `SwapClient.RECONNECT_INTERVAL` here?

Reply:
It sure would, good thinking.